python/tests/runtime_aggtest/illarg_tests/test_agg.py

```diff
@@ -0,0 +1,194 @@
+                      EVERY(intt) AS intt
+                      FROM illegal_tbl"""
+        self.expected_error = "Cannot apply 'EVERY' to arguments of type 'EVERY(<INTEGER>)'. Supported form(s): 'EVERY(<BOOLEAN>)'"
+
+
+# SOME
+class illarg_some_illegal(TstView):
+    def __init__(self):
+        # checked manually
+        self.sql = """CREATE MATERIALIZED VIEW illarg_some_illegal AS SELECT
+                      SOME(intt) AS intt
+                      FROM illegal_tbl"""
+        self.expected_error = "Cannot apply 'SOME' to arguments of type 'SOME(<INTEGER>)'. Supported form(s): 'SOME(<BOOLEAN>)'"
+
+
+# BOOL_AND
+class illarg_bool_and_illegal(TstView):
+    def __init__(self):
+        # checked manually
+        self.sql = """CREATE MATERIALIZED VIEW illarg_bool_and_illegal AS SELECT
+                      BOOL_AND(udt[2] IS NOT NULL, arr[2] IS NOT NULL) AS udt
+                      FROM illegal_tbl"""
+        self.expected_error = "Invalid number of arguments to function 'BOOL_AND'. Was expecting 1 arguments"
+
+
+# BOOL_OR
+class illarg_bool_or_illegal(TstView):
+    def __init__(self):
+        # checked manually
+        self.sql = """CREATE MATERIALIZED VIEW illarg_bool_or_illegal AS SELECT
+                      BOOL_OR(arr[2] IS NOT NULL, arr[2] IS NOT NULL) AS arr
+                      FROM illegal_tbl"""
+        self.expected_error = "Invalid number of arguments to function 'BOOL_OR'. Was expecting 1 arguments"
+
+
+# MAX
+class illarg_max_illegal(TstView):
+    def __init__(self):
+        # checked manually
+        self.sql = """CREATE MATERIALIZED VIEW illarg_max_illegal AS SELECT
+                      MAX(udt[2], arr[2]) AS udt
+                      FROM illegal_tbl"""
+        self.expected_error = (
+            "Invalid number of arguments to function 'MAX'. Was expecting 1 arguments"
+        )
+
+
+# MIN
+class illarg_min_illegal(TstView):
+    def __init__(self):
+        # checked manually
+        self.sql = """CREATE MATERIALIZED VIEW illarg_min_illegal AS SELECT
+                      MIN(udt[2], arr[2]) AS udt
+                      FROM illegal_tbl"""
+        self.expected_error = (
+            "Invalid number of arguments to function 'MIN'. Was expecting 1 arguments"
+        )
+
+
+# SUM
+class illarg_sum_illegal(TstView):
+    def __init__(self):
+        # checked manually
+        self.sql = """CREATE MATERIALIZED VIEW illarg_sum_illegal AS SELECT
+                      SUM(udt) AS udt
+                      FROM illegal_tbl"""
+        self.expected_error = "Cannot apply 'SUM' to arguments of type "
+
+
+class illarg_sum_illegal1(TstView):
+    def __init__(self):
+        # checked manually
+        self.sql = """CREATE MATERIALIZED VIEW illarg_sum_illegal1 AS SELECT
+                      SUM(str) AS str
+                      FROM illegal_tbl"""
+        self.expected_error = "Cannot parse hello  into a DECIMAL(38, 19)"
```

Comment:
this test is different, it fails at runtime, not at compilation.
Does the test differentiate between the two?